pkgs/applications/graphics/litemdview/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, fetchFromGitea
+, fetchpatch
+, gtkmm3
+, autoconf
+, automake
+, libtool
+, cairo
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "litemdview";
+  # litemdview -v
+  version = "0.0.32";
+
+  src = fetchFromGitea {
+    domain = "notabug.org";
+    owner = "g0tsu";
+    repo = "litemdview";
+    rev = "eacbbc837e";
+    hash = "sha256-sD0y+/7TogywrvftxEXT1EyBf3BXH9QWbawFVOF8xRc=";
+  };
+
+  buildInputs = [
+    gtkmm3
+  ];
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    pkg-config
+    libtool
+    cairo
+  ];
+
+  buildPhase = "
+    ./bootstrap
+    ./configure --prefix=$out
```

Comment:
Not sure what the `./bootstrap` does, though `./configure` is traditionally ran during `configurePhase`, and might already happen by the default phase as provided by `stdenv.mkDerivation`.

Please check the necessity of these being need to be run explicitely and move them to the correct phase.